packages/database/src/migrations/20210823032403-LESMISChildrenOverAgeForGradeVisual-modifies-data.js

```diff
@@ -0,0 +1,184 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, findSingleRecordBySql } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const GENDER_CODES = ['m', 'f', 't'];
+
+const generateReportConfig = (entityLevel, educationLevel, name) => ({
+  fetch: {
+    dataElements: GENDER_CODES.map(gender => `oafg_${entityLevel}_${educationLevel}_${gender}`),
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+      },
+    ],
+  },
+  transform: [
+    'keyValueByDataElementName',
+    {
+      transform: 'select',
+      "'Male'": `exists($row.oafg_${entityLevel}_${educationLevel}_m) ? $row.oafg_${entityLevel}_${educationLevel}_m/100 : undefined`,
+      "'Female'": `exists($row.oafg_${entityLevel}_${educationLevel}_f) ? $row.oafg_${entityLevel}_${educationLevel}_f/100 : undefined`,
+      "'Total'": `exists($row.oafg_${entityLevel}_${educationLevel}_t) ? $row.oafg_${entityLevel}_${educationLevel}_t/100 : undefined`,
+      "'name'": `'${name}'`,
+    },
+    {
+      transform: 'aggregate',
+      name: 'group',
+      '...': 'last',
```

Comment:
I'm interested in the idea of a `only` aggregation function, that asserts there was only one of the field across all rows being aggregated. Out of scope for this PR, but as a discussion point, what do you reckon?